images/elasticsearch/es-curl

```diff
@@ -11,4 +11,4 @@ COMMAND=$2
 shift
 shift
 
-curl -X $VERB -k -u "admin:$LOGSDB_ADMIN_PASSWORD" "http://localhost:9200/$COMMAND" -H 'Content-Type: application/json' "$@"
\ No newline at end of file
+curl -X $VERB -k -u "admin:$LOGSDB_ADMIN_PASSWORD" "http://localhost:9200/$COMMAND" -H 'Content-Type: application/json' "$@"
```

Comment:
see https://github.com/amazeeio/lagoon/issues/1877#issuecomment-628023843
this is unfortunately not gonna work

Reply:
I tried on my local to start an elasticsearch container and run some `es-curl` commands without errors. Should I fix the code as well?

Reply:
I added a check in the last commit. If `$LOGSDB_ADMIN_PASSWORD` variable exists, then es-curl will use authentication, otherwise not